external/model-preparation-algorithm/mpa_tasks/apis/task.py

```diff
@@ -80,7 +80,7 @@ def _run_task(self, stage_module, mode=None, dataset=None, parameters=None, **kw
             train_data_cfg = Stage.get_train_data_cfg(self._data_cfg)
             train_data_cfg['data_classes'] = data_classes
             new_classes = np.setdiff1d(data_classes, model_classes).tolist()
-            train_data_cfg['old_new_indices'] = self._get_old_new_indices(dataset, new_classes)
+            train_data_cfg['new_classes'] = new_classes
```

Comment:
According to the current PR code, classification does not remove 'new_classes' from kwargs, unlike det/seg dataset, 
but can it be removed elsewhere? Is there a case where this conflicts with other codes?

Reply:
There were no conflicts with other codes.